src/Plugins/OrientationAnalysis/src/OrientationAnalysis/Filters/Algorithms/ComputeKernelAvgMisorientations.cpp

```diff
@@ -68,13 +70,11 @@ class FindKernelAvgMisorientationsImpl
 
       if(counter > increment)
       {
-        auto now = std::chrono::steady_clock::now();
-        if(std::chrono::duration_cast<std::chrono::milliseconds>(now - start).count() > 1000)
-        {
-          m_Filter->sendThreadSafeProgressMessage(counter);
-          counter = 0;
-          start = std::chrono::steady_clock::now();
-        }
+        progressMessenger.sendProgressMessage(counter, [&](usize currentProgress, usize maxProgress) {
```

Comment:
The original code funneled the messages through the filter class in order to obtain an accurate count of how much has been computed and to throttle how fast messages can be sent. This implementation does not retain that functionality. I now get a large number of messages to the output. And while these messages do not show up on a time trace, the shear number of messages is too much.

Reply:
@JDuffeyBQ I Slacked the Google Drive link to you